src/WebJobs.Extensions.Twilio/Config/TwilioExtensionConfigProvider.cs

```diff
@@ -83,10 +83,8 @@ internal static CreateMessageOptions CreateMessageOptions(JObject messageOptions
         {
             var options = new CreateMessageOptions(new PhoneNumber(GetValueOrDefault<string>(messageOptions, "to")))
             {
-                ProviderSid = GetValueOrDefault<string>(messageOptions, "providerSid"),
```

Comment:
Looks like a number of new properties have been added as well:

![Screenshot 2023-01-04 at 2 22 15 PM](https://user-images.githubusercontent.com/2507935/210661221-818e1665-6ac1-4d3e-81e8-263f4707ea69.png)

Would it make sense for those to be supported here?

Reply:
I added them just in case.